src/modules/claims/claims.service.ts

```diff
@@ -1563,9 +1574,9 @@ export class ClaimsService {
         errors: [ERROR_MESSAGES.NO_CLAIM_RESOLVED],
       };
     }
-    const credentialIsOffChain = resolvedCredential.eip191Jwt;
+    const credentialIsOffChain = isEIP191Jwt(resolvedCredential);
```

Comment:
Is it possible to do without extra variable and casting, like so:
```
if (isEIPjwt(resolvedCredential) {
   return this.verifyRoleEIP191JWT(resolvedCredential);
} 
else if isVerifiableCredential(resolvedCredential) {
   return this.verifyVc(resolvedCredential);
}
```

Reply:
I think this would increase if conditions for no reasons, I could move this to ternary operation like 
`return isEIP191Jwt(resolvedCredential) ? this.verifyRoleEIP191JWT(resolvedCredential)`